pyiron_continuum/damask/damaskjob.py

```diff
@@ -1,66 +1,166 @@
+# Reference : https://github.com/jan-janssen/pyiron-damask/blob/master/damaskjob.py
 from pyiron_base import Project, GenericJob, GenericParameters
+from pyiron_base import InputList 
 import numpy as np
 import matplotlib.pyplot as plt
 from damask import Config
-from damask import Grid
+from damask import Geom
 from damask import Result
 from damask import seeds
+import pyvista as pv
 import h5py
 import yaml
 import os
+import warnings
 
 
 class DAMASKjob(GenericJob):
     def __init__(self, project, job_name):
         super(DAMASKjob, self).__init__(project, job_name)
-        self.input = GenericParameters(table_name="input")
-        self.input['C11'] = 0
-        self.input['C12'] = 0
-        self.input['C44'] = 0
-        self.input['grid'] = np.array([16,16,16])
-        self.input['size'] = np.array([1.0,1.0,1.0])
-        self.input['grains'] = 20
-        self.input['cores'] = 4
-        self.executable = "mpiexec -np "+str(self.input['cores'])+" DAMASK_grid -l tensionX.load -g damask.vtr"
+        self.input = InputList()
+        self.executable = self.executable_activate
```

Comment:
@Ahmed-Aslam, why did not you just use: `self._executable_activate()`. When I checked other job like [lammpsjob](https://github.com/pyiron/pyiron_atomistics/blob/c5df5e87745d7b575463f7b2a0b588e18007dc40/pyiron_atomistics/lammps/lammps.py#L37), it should be sufficient. As mentioned by Jan, by this, pyiron looks for the executable in the resource directory.

Reply:
I thought this was just required to search script job that should be link to executable. In the recent change it was done, but not in the same way as with other pyiron jobs, I'll update it.